app/Http/Controllers/Api/MoneyController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\src\Deviation\Deviation;
+use App\src\RequestMoneyValidator\Dto\TransactionDto;
+use App\src\RequestMoneyValidator\Requests\RequestMoneyValidatorRequest;
+use App\src\RequestMoneyValidator\Services\RequestMoneyValidator;
+use Illuminate\Http\JsonResponse;
+use InvalidArgumentException;
+use Symfony\Component\HttpFoundation\Response;
+use Throwable;
+
+class MoneyController extends Controller
+{
+    public function __construct(private readonly Deviation $deviation)
+    {
+    }
+
+    final public function check(RequestMoneyValidatorRequest $request): JsonResponse
```

Comment:
The ticket was not supposing to do this extra work of covering the endpoint and db models. But good job anyway